react-graphql-apollo-tutorial.md

```diff
@@ -208,22 +206,22 @@ Basically that's it for sending a query with Apollo Client. As noted before, Apo
+More meta information about the request can be found next to the `data` object. It shows whether the data is still loading, as well as specific details about the {{% a_blank "network status" "https://github.com/apollographql/apollo-client/blob/master/packages/apollo-client/src/core/networkStatus.ts" %}}, and see whether the requested data is stale on the server-side.
 
 ### Exercises:
 
-* explore GitHub's GraphQL API
-  * get comfortable navigating through their documentation
-  * add other fields for the `organization` field
-* read more about {{% a_blank "why you should use Apollo Client" "https://www.apollographql.com/docs/react/why-apollo.html" %}}
-* read more about the {{% a_blank "networkStatus property and its possible values" "https://github.com/apollographql/apollo-client/blob/master/packages/apollo-client/src/core/networkStatus.ts" %}}
-* invest 3 minutes of your time and take the {{% a_blank "quiz" "https://www.surveymonkey.com/r/5MF35H5" %}}
+* Explore GitHub's GraphQL API
+  * Get comfortable navigating through their documentation
+  * Add other fields for the `organization` field
+* Read more about {{% a_blank "why you should use Apollo Client" "https://www.apollographql.com/docs/react/why-apollo.html" %}}
+* Read more about the {{% a_blank "networkStatus property and its possible values" "https://github.com/apollographql/apollo-client/blob/master/packages/apollo-client/src/core/networkStatus.ts" %}}
+* Invest 3 minutes of your time and take the {{% a_blank "quiz" "https://www.surveymonkey.com/r/5MF35H5" %}}
 
 {{% chapter_header "Apollo Client with Pagination, Variables, Nested Objects and List Fields" "graphql-apollo-client-pagination" %}}
 
-You have learned about GraphQL pagination and other GraphQL features in previous sections where you have built the React with GraphQL application without Apollo. This section will introduce a couple of these features, but keeps most of them for you as your exercise to internalize these things. Let's start this section with introducing GraphQL variables. The `login` argument for the organization field in the previous query can be substituted with such a variable. First, you have to introduce the variable in your GraphQL query:
+You learned about GraphQL pagination and other GraphQL features in previous sections when you built the React with GraphQL application without Apollo. This section will introduce a couple of these features. This section wil introduce GraphQL variables. The `login` argument for the organization field in the previous query can be substituted with such a variable. First, you have to introduce the variable in your GraphQL query:
```

Comment:
I changed this because it wasn't a good sentence.  It has a lot of extra words that don't add anything useful.  Specifically "....but keeps most of them for you as your exercise to internalize these things..."  That's telling the reader "I am keeping things hidden so you can figure it out for yourself" which is the opposite of what a tutorial should do, so it's best left unsaid.  Also, you don't always need to be elaborate when you're telling a reader what's coming up next.  Let them read on and see for themselves what's there, and they will see in the actual exercises what you want them to gather on their own.

Reply:
Agreed. Thanks for clarification! 👍